BootloaderCorePkg/Library/PciEnumerationLib/PciEnumerationLib.c

```diff
@@ -1730,27 +1729,24 @@ PciScanRootBridges (
       Bus = Index;
     }
 
-    Address = PCI_EXPRESS_LIB_ADDRESS (Bus, 0, 0, 0);
-    if (PciExpressRead16 (Address) != 0xFFFF) {
```

Comment:
This diff looks complicated, but really it's only removal of these two lines, the closing brace, and the Address declaration. The rest is just indent removal.